app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { fetchGreetingData } from "../redux/actions/greeting-action";
+
+const Greeting = () => {
+  const greetings = useSelector((state) => state.greetingSlice.greetings);
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchGreetingData());
+  }, [dispatch]);
+
+  const getRandomGreeting = () => {
+    if (greetings.length === 0) {
+      return null;
+    }
+    const randomIndex = Math.floor(Math.random() * greetings.length);
+    return greetings[randomIndex];
+  };
+
+  const greeting = getRandomGreeting();
+
+  return (
+    <div>
+      {greeting && <h1>{greeting}</h1>}
+      <button onClick={() => dispatch(fetchGreetingData())}>
+        Get a Greeting
+      </button>
+    </div>
+  );
```

Comment:
- [ ] Good job trying to fix changes required by previous reviewer. However, as the major requirement of this project is to ensure that **the two apps work together** which is not the case of your project. Right now, I can see the API working perfectly but your react app display a blank page with some errors in the console. I recommend you fix that as it is the **main requirement for the two apps to work together**. Below is how your react application looks like:
![Screenshot from 2023-06-06 17-42-43](https://github.com/Lancelot-SO/hello-rails-react/assets/83042742/cafd833c-a909-4e6c-afd7-60834bd0900d)

Reply:
Can I delete this repo and redo the project again I think my installation setup wasn't right and I have exhausted all my reviews